proto/storage/cluster.proto

```diff
@@ -341,5 +347,30 @@ message AdmissionControlHealthInfo {
 
     // Collection of errors that occurred while trying to obtain admission control health info.
     repeated string status_errors = 3;
+}
+
+// ScannerHealthInfo carries data about scanner deployment but does not include scanner health status
+// derived from this data.
+// Aggregated scanner health status is not included because it is derived in central and not in the component that
+// first reports ScannerHealthInfo (sensor).
+message ScannerHealthInfo {
```

Comment:
Is this also solely for the "local" Scanner? If so, can you add that comment here?

Reply:
Comment added.